help_to_heat/templates/frontdoor/feedback.html

```diff
@@ -12,72 +12,110 @@
   {% endif %}
 
 <h1 class="govuk-heading-l">{{_("Help us improve the service")}}</h1>
-<form id="feedback" method="post" novalidate>
+<form id="feedback" method="post">
     {{csrf_input}}
 
+    {% if errors %}
+    <div class="govuk-error-summary" data-module="govuk-error-summary">
+      <div role="alert">
+        <h2 class="govuk-error-summary__title">
+          {{_("There is a problem")}}
+        </h2>
+        <div class="govuk-error-summary__body">
+          <ul class="govuk-list govuk-error-summary__list">
+            <li>
+              <a href="#">{{_("Please answer at least one question before submitting feedback")}}</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+
+    <div class="govuk-form-group govuk-!-padding-bottom-7">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-label-wrapper">
```

Comment:
Looks like yours isn't the original sin here, but we shouldn't have multiple `h1` tags in a page. All these questions should probably be h2.

Reply:
Got it, thanks, changed the questions to h2!